Pod/Classes/SuperMock.swift

```diff
@@ -28,7 +28,23 @@ public class SuperMock: NSObject {
         NSURLSession.sharedSession().configuration.protocolClasses?.append(SuperMockURLProtocol)
         
         SuperMockResponseHelper.bundleForMocks = bundle
+        
+    }
+    
+    class func beginRecording(bundle: NSBundle?, mocksFile: String? = "Mocks.plist", policy: SuperMockResponseHelper.RecordPolicy) {
+        
+        NSURLProtocol.registerClass(SuperMockRecordingURLProtocol)
+        NSURLSessionConfiguration.defaultSessionConfiguration().protocolClasses = [SuperMockRecordingURLProtocol.self]
+        NSURLSession.sharedSession().configuration.protocolClasses?.append(SuperMockRecordingURLProtocol)
+        
+        SuperMockResponseHelper.bundleForMocks = bundle
+        SuperMockResponseHelper.sharedHelper.recording = true
+        
+    }
```

Comment:
Yep you are right

> On 7 Dec 2015, at 09:38, grzegorzleszek <notifications@github.com> wrote:
> 
> In Pod/Classes/SuperMock.swift <https://github.com/michaelarmstrong/SuperMock/pull/3#discussion_r46801040>:
> 
> > @@ -28,7 +28,23 @@ public class SuperMock: NSObject {
> >          NSURLSession.sharedSession().configuration.protocolClasses?.append(SuperMockURLProtocol)
> >          
> >          SuperMockResponseHelper.bundleForMocks = bundle
> > +        
> > +    }
> > +    
> > +    class func beginRecording(bundle: NSBundle?, mocksFile: String? = "Mocks.plist", policy: SuperMockResponseHelper.RecordPolicy) {
> > +        
> > +        NSURLProtocol.registerClass(SuperMockRecordingURLProtocol)
> > +        NSURLSessionConfiguration.defaultSessionConfiguration().protocolClasses = [SuperMockRecordingURLProtocol.self]
> > +        NSURLSession.sharedSession().configuration.protocolClasses?.append(SuperMockRecordingURLProtocol)
> > +        
> > +        SuperMockResponseHelper.bundleForMocks = bundle
> > +        SuperMockResponseHelper.sharedHelper.recording = true
> > +        
> > +    }
> I think beginRecording should be public, so it could be used in obj-c. (same for endRecording)
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/michaelarmstrong/SuperMock/pull/3/files#r46801040>.
>